core/utils.go

```diff
@@ -19,6 +19,32 @@ func Keccak256(data []byte) ([32]byte, error) {
 	return digest, nil
 }
 
+func HashWithPrefix(prefix []byte, data []byte) ([32]byte, error) {
+	prefixHash, err := Keccak256(prefix)
```

Comment:
Keccak shalln't return an error.

Reply:
Implementation also doesn't

Reply:
Indeed - checking this out

Reply:
Done in [e9c4928](https://github.com/NethermindEth/near-sffl/pull/104/commits/e9c4928ff7f1c5b675af7c592a98376a0f50d085)